1.8/usage/service-discovery/haproxy-adminrouter.md

```diff
@@ -0,0 +1,69 @@
+---
+post_title: Configuring HTTP Proxy in Front of Admin Router
+nav_title: Configure HTTP Proxy
+menu_order: 61
+---
+
+You can configure a secure communication channel from a custom hostname and port to DC/OS Admin Router by using an HTTP Proxy.
+ 
+The HTTP Proxy must perform on-the-fly HTTP request and response header modification because DC/OS is not aware of the custom hostname and port that is being used by user agents to address the HTTP proxy.
+
+These instructions provide a tested [HAProxy](http://www.haproxy.org/) configuration example that handles the named request/response rewriting. This example ensures that the communication between HAProxy and DC/OS Admin Router is TLS-encrypted. It uses the DC/OS [CA certificate](/1.8/administration/tls-ssl/) so that HAProxy can verify the TLS certificate presented by Admin Router.
```

Comment:
A reasonable simplification that renders this introduction non-specific for Open or EE: we can remove the last sentence here.

That is, change
```
This example ensures that [...] is TLS-encrypted. It uses the DC/OS [CA certificate](/1.8/administration/tls-ssl/) so that HAProxy can verify the TLS certificate presented by Admin Router.
```
to
```
This example ensures that [...] is TLS-encrypted.
```